wesnoth-help.po

```diff
@@ -4083,8 +4082,8 @@ msgid ""
 msgstr ""
 "\n"
 "\n"
-"Konečně, vesnice vystavěné na vodě jsou obvykle nevhodné pro pozemní "
-"jednotky a neposkytují bonusy ani obranné, ani v pohybu, které jsou pro "
+"Konečně, vesnice vystavěné na vodě jsou obvykle pro pozemní jednotky "
+"nehostinné a neposkytují bonusy ani obranné, ani v pohybu, které jsou pro "
```

Comment:
nechci to moc přeorávat, ale ta předložka **v** mi tam zní cize; pokud vám ne, bojovat s vámi nechci
další možnost: **při pohybu**
```suggestion
"nehostinné a neposkytují bonusy ani obranné, ani pro pohyb, které jsou pro "
```